components/states/DiverNotCertifiedState.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import Image from "next/image";
+import { useActor } from "@xstate/react";
+
+import { STATE_ACTIONS } from "../../utils/state-machine";
+import StatePage from "./shared/StatePage";
+import StateTitle from "./shared/StateTitle";
+import StateCards from "./shared/StateCards";
+import CardComponent from "./shared/CardComponent";
+import { MyContext } from "../ReservationController";
+
+const DiverNotCertifiedState = () => {
+  const machine = React.useContext(MyContext);
+  const [state, send] = useActor(machine);
+
+  return (
+    <StatePage>
+      <StateTitle title="You must be certified to book a recreational dive!" />
+      <StateCards>
+        <CardComponent
+          icon={
+            <Image
+              src="/icons/certificate.svg"
+              alt=""
+              width={150}
+              height={150}
+            />
+          }
+          text="Get certified with us"
+          onClick={() =>
+            send({
+              type: STATE_ACTIONS.CERTIFICATION_DIVE,
+              value: "certification",
+              previousState: STATE_ACTIONS.RESERVATION,
```

Comment:
It may seem strange that we are doing this - but we don't want Divers to be able to return to this state from the `CertificationDive` state. When Divers get to the `CertificationDive` state we want the back button on that state to take them back to the initial reservation - this will simplify the logic for us a lot and it also makes more sense (we can start running into some situations where the context values just don't make sense if we allow the user to return to the `IsDiverCertified` state. You can test out the flow to see what I mean.

Reply:
In the `Reservation` state - we will reset all of the context values that have been modified up to that point ( you can check the actions on the state-machine)

Reply:
yeah this makes sense!

Reply:
we also need to clear `previousState` when we arrive to `RESERVATION` state